src/usr/local/www/vpn_ipsec_mobile.php

```diff
@@ -231,6 +236,12 @@
 		}
 	}
 
+	if ($pconfig['radius_ip_priority_enable']) {
+		if (!(isset($mobileph1) && $mobileph1['authentication_method'] == 'eap-radius')) {
```

Comment:
Second test on this line needs parenthesis

Reply:
fixed